### PR TITLE
Restore MBR vest and remove ESAPI vest

### PR DESCRIPTION
**Pull request by Firestorm01X2**
Restore MBR vest and remove ESAPI vest

#### Purpose of change
Fixes https://github.com/cataclysmbnteam/Cataclysm-BN/issues/111

#### Describe the solution
Remove unbalanced ESAPI vest and resotre old balanced MBR vest variants.

- Revert: https://github.com/CleverRaven/Cataclysm-DDA/pull/36920
- Obsolate: ESAPI vest
- Remove Kevlar MBR. We are ptetending that empty MBR is Kevlar MBR already. So buff emepty MBR vest
- emtpy MBR vest in not craftable
- Migrate ESAPI vests and plates
- Do replacing over item groups, npcs and professions.

#### Describe alternatives you've considered
None.

#### Testing
1) Start the game
2) Make sure that MBR verst not in game and ESAPI vest obsolated.

#### Additional context
None.

### CODE REVIEW

File: data/json/recipes/armor/torso.json [hunk 1]
```diff
@@ -488,20 +488,86 @@
         [ "fanny", 1 ]
       ],
       [ [ "duct_tape", 150 ] ],
-      [ [ "kevlar", 1 ], [ "ballistic_vest_empty", 1 ], [ "swat_armor", 1 ], [ "kevlar_plate", 16 ] ]
+      [ [ "kevlar", 1 ], [ "modularvest", 1 ], [ "swat_armor", 1 ], [ "kevlar_plate", 16 ] ]
     ]
   },
   {
-    "result": "ballistic_vest_esapi",
+    "result": "modularvest",
```
Comment (Coolthulhu) [inline data/json/recipes/armor/torso.json:495]: Not sure if it should be craftable. It's probably fine without the recipe.

File: data/json/recipes/recipe_obsolete.json [hunk 2]
```diff
@@ -2454,5 +2424,18 @@
...
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_TORSO",
+    "skill_used": "tailor",
+    "time": "2 m",
+    "reversible": true,
+    "decomp_learn": 0,
+    "autolearn": true,
+    "components": [ [ [ "ballistic_vest_empty", 1 ] ], [ [ "esapi_plate", 2 ] ], [ [ "esbi_plate", 2 ] ] ],
+    "flags": [ "BLIND_EASY" ]
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_obsolete.json:2439]: The recipe should be marked as obsolete and not contain all the data. See other recipes in the file.

**Comment by Firestorm01X2** [comment]
Remove Kevlar MBR and its recepie. Buf empty MBR with Kevlar MBR stats (do not forget encumbrance.

File: data/json/recipes/armor/torso.json (hunk 1 above)
Comment (Firestorm01X2) [reply data/json/recipes/armor/torso.json]: Done

File: data/json/recipes/recipe_obsolete.json (hunk 2 above)
Comment (Firestorm01X2) [reply data/json/recipes/recipe_obsolete.json]: Done

**Comment by Firestorm01X2** [comment]
Military recruit still using emetpy MBR for balance reasons. Emtpy MBR now buffed a bit anyway.

**Comment by Firestorm01X2** [comment]
It is done.

**Comment by Firestorm01X2** [comment]
Just a note. Empty MBR vest was buffed not that much for consistency with other vests.

**Comment by Coolthulhu** [comment]
Minor things to fix in the future:
* MBR is a Kevlar pinata - it can be cut up for 23 plates
* It has a holster ammo action, which is a needless detail. Could lose that one.
* Ceramic variant could gain 2-3 points of encumbrance

**Comment by Firestorm01X2** [comment]
>     * It has a holster ammo action, which is a needless detail. Could lose that one.

tac vest has it also. And XL Kevlar vest. So consistency wise it make sense.

**Comment by Coolthulhu** [comment]
Tac vest is not armor, it's an ammo carrying vest.
XL Kevlar vest is a weird item - it has stats too bad to be used.
The XL vest could use both a buff in armor vs encumbrance and also to lose that holster thing.

**Pull request merged**